### DISCUSSION TITLE
What is the maximum data storage capacity in the Catalyst's Hub? / ¿Cuál es la capacidad máxima de almacenamiento en un hub de Catalyst?

### DISCUSSION THREAD

**Discussion by SLLDeC** [czi-catalystproject/czi-catalystproject.github.io#39] [Q&A]
Previous answers on Slack:

@jnywong said:
> Every hub user gets a "home directory" to store files and the quota is limited to 10 GB.

@jmunroe asked:
>What kind of data? Is the data already publicly available? Is object storage an option or does it require a block-file system?  Is this temporary, scratch space or for persistent long-term storage? Does the data need to be accessible outside of the JupyterHub?

**Comment by SLLDeC** [comment]
- What kind of data?
Human gut metagenome data
- Is the data already publicly available?
No, it is not publicly available.
- Is object storage an option or does it require a block-file system?
I didn't know the difference between Object Storage and Block File. But as far as I can see, we would need Block File type because they are large data files.
- Is this temporary, scratch space or for persistent long-term storage?
As in a year after we finish processing them, they can be deleted. Not when the user logs out. We will be several users.
- Does the data need to be accessible outside of the JupyterHub?
Not if the Jupyter hub allows us to install other Linux tools; if not, yes. We usually connect via SSH to Ubuntu.

--------------------------------------------------------------------------------------------------------------------------

- ¿Qué tipo de datos necesitan almacenar?
Datos de metagenomas de intestino humano
- ¿Son datos que ya están disponibles públicamente?
No, no están disponibles púbicamente
- ¿Les sería útil un object storage o requieren un sistema block-file? 
No sabía la diferencia entre Object Storage y Block File. Pero por lo que puedo ver necesitaríamos de tipo Block File por ser archivos de datos grandes.
- El almacenamiento es temporario? Cuando se puede borrar? Cuando el usuario cierra sesión o nunca?
Como en un año que acabemos de procesarlos se pueden borrar. No cuando el usuario cierra sesión. Seremos varios usuarios.
- ¿Necesitan poder acceder a estos datos por fuera del JupyterHub?
Si el Jupyter hub nos deja instalar otras herramientas para linux no, si no se puede si. Usualmente nos contectamos por ssh a ubuntu

**Comment by jnywong** [comment]
Okay thanks for the further details!

> What is the maximum data storage capacity in the Catalyst's Hub?

Technically the capacity is unlimited since we are using cloud resources, and the following answer to...

> Is object storage an option or does it require a block-file system?
I didn't know the difference between Object Storage and Block File. But as far as I can see, we would need Block File type because they are large data files.

... makes me think that cloud object storage would be suitable. Again, I think the answer is that this is technically possible but not supported by default. 

> Does the data need to be accessible outside of the JupyterHub?
Not if the Jupyter hub allows us to install other Linux tools; if not, yes. We usually connect via SSH to Ubuntu.

Do you know what Linux tools are required?
